src/tests/topp/THIRDPARTY/third_party_tests.cmake

```diff
@@ -356,3 +360,10 @@ if (NOT (${SPECTRAST_BINARY} STREQUAL "SPECTRAST_BINARY-NOTFOUND") AND FALSE)
   set_tests_properties("TOPP_SpectrastSearchAdapter_1_out" PROPERTIES DEPENDS "TOPP_SpectrastSearchAdapter_1")
 endif()
 
+if (NOT (${LUCIPHOR_BINARY} STREQUAL "LUCIPHOR_BINARY-NOTFOUND"))
+  ### NOT needs to be added after the binarys have been included
```

Comment:
Copy-paste from crux adapter? I don't even know what it means there. Any idea?

Reply:
But let the tests run through once first before making another change.

Reply:
yes, copy and paste error!